Misc/NEWS.d/next/Library/2024-09-09-12-48-37.gh-issue-123853.e-zFxb.rst

```diff
@@ -0,0 +1,3 @@
+Update the table of Windows language code identifiers (LCIDs) used by
+:func:`locale.getdefaultlocale()` on Windows to protocol version 16.0
+(4/23/2024).
```

Comment:
```suggestion
(2024-04-23).
```

Reply:
This is how it is indicated in the original document.